src/card.css

```diff
@@ -0,0 +1,71 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: stretch;
+  width: 200px;
+  padding: var(--card-padding);
+  border: var(--card-border);
+  border-radius: var(--card-border-radius);
+  background-color: var(--card-background-color);
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  display: block;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  margin: 0 0 4px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: var(--text-accent-color);
+}
+
+.card__product-code {
+  margin-bottom: 16px;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 40px;
+  background-color: var(--main-color);
+  border: 1px solid var(--main-color);
+  border-radius: var(--card-border-radius);
+  font-size: 14px;
+  line-height: 16px;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #fff;
+  font-weight: 700;
+}
+
+.card__button:hover {
+  background-color: var(--card-background-color);
+  color: var(--main-color);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.card__pice-text {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
```

Comment:
It's better to use `line-height` in a percentage or relative value: `line-height: 1.5;`